database/postgres/index.js

```diff
@@ -0,0 +1,57 @@
+const { Client } = require('pg');
+
+const client = new Client({
+  user: 'Li',
+  host: 'localhost',
+  password: 'pw',
+  database: 'kickstarter',
+  port: 5432,
+});
+
+client.connect();
+
+
+const func = {};
+
+func.findUpdates = (projectId, callback) => {
+  const query = `SELECT * FROM updates WHERE projectid = ${projectId}`;
+  client.query(query)
+    .then((results) => callback(results))
```

Comment:
Invoking a callback inside a Promise seems redundant to me. You should return client.query(query) instead so that the caller can do something with the result.

Reply:
You're not using the PG promises package so this is irrelevant.